prisma/schema.prisma

```diff
@@ -9,27 +9,117 @@ generator client {
 }
 
 generator prismaClassGenerator {
-    provider = "prisma-class-generator"
-    dryRun = false
+  provider = "prisma-class-generator"
+  dryRun   = false
 }
 
 datasource db {
   provider = "postgresql"
   url      = env("POSTGRES_URL")
 }
 
-model User {
-  id    Int     @id @default(autoincrement())
-  email String  @unique
-  name  String?
-  posts Post[]
+model Path {
+  id                  Int                   @id @default(autoincrement())
+  title               String
+  description         String
+  courses             Course[]
+  studentPathInstance StudentPathInstance[]
 }
 
-model Post {
-  id        Int     @id @default(autoincrement())
-  title     String
-  content   String?
-  published Boolean @default(false)
-  author    User    @relation(fields: [authorId], references: [id])
-  authorId  Int
+model PathInstance {
+  id                            Int                   @id @default(autoincrement())
+  title                         String
+  description                   String
+  date_from                     DateTime
+  date_to                       DateTime
+  number_of_students            Int
```

Comment:
I remember we decided the in prisma we do a kamal case 
like numberOfStudents

I will take care of this